ratslam_view_template:
  ros__parameters:
    image_crop_y_max: 150
    template_x_size: 60
    template_y_size: 20
    vt_shift_match: 4
    vt_step_match: 1
    vt_match_threshold: 0.03
    # draw
    enable: true
    vt_window_width: 416
    vt_window_height: 480

ratslam_pose_cells:
  ros__parameters:
    pose_cell:
      pc_dim_xy: 30
      pc_dim_th: 36
      pc_w_e_dim: 7
      pc_w_i_dim: 5
      pc_w_e_var: 1
      pc_w_i_var: 2
      pc_global_inhib: 0.00002
      vt_active_decay: 1.0
      pc_vt_inject_energy: 0.15
      pc_cell_x_size: 1.0
      exp_delta_pc_threshold: 2.0
      pc_vt_restore: 0.05
    draw:
      enable: true
      vt_window_width: 416
      vt_window_height: 480

ratslam_experience_map:
  ros__parameters:
    experience_map:
      exp_correction: 0.5
      exp_loops: 1
      exp_initial_em_deg: 90.0
      max_goals: 10
    draw:
      enable: true
      vt_window_width: 416
      vt_window_height: 480
